pkg/registry/core/seed/strategy_test.go

```diff
@@ -51,4 +55,72 @@ var _ = Describe("Strategy", func() {
 			Expect(newSeed.Generation).To(Equal(oldSeed.Generation + 1))
 		})
 	})
+
+	Describe("#Validate", func() {
```

Comment:
I think this tests should be in `pkg/apis/core/validation/utils_test.go`

Reply:
The function is defined in the package file `strategy.go` and I followed the example of `pkg/registry/core/shoot/strategy_test.go` where `Canonicalize` was declared in `shoot/stratergy.go`and the tests were written in `stratergy_test.go`

Reply:
But the function inside Canonicalize is not exposed, which is not in the case of `ValidateSeedCreationWithOldDependencyWatchdogSettings`.